### ISSUE TITLE
1.59 Issues

### ISSUE THREAD

**Issue by bearoso** [snes9xgit/snes9x#498]
Anything that should hold up a 1.59 release goes here.

* @OV2, I'd like to change the Windows port's title. Following UX guidelines, it would show the name of the loaded game, followed by the application name. This would use the ROM image file's name, not the game title in the header, which is often buggy and incomplete. So we'd load "Super Mario World.sfc" and have:
`Super Mario World - Snes9x`
Making sure to make the final x lowercase and fix that once-and-for-all. (https://www.youtube.com/watch?v=IjmtVKOAHPM)
Just checking that it's OK with you; don't feel obligated to work on anything I suggest.

* I'm planning on moving the GTK Display Settings into some sub-tabs to eliminate scrolling.

* There's a mention of some stuttering on the Snes9x.com forums that's probably DWM compositor related. We can smooth it out by putting a dwmflush after the buffer swap in GL to avoid the race condition, and that means dynamically loading the function and checking Windows version. Any alternatives are in DXGI and we'd have to rewrite the buffer backends for that.

**Comment by OV2** [comment]
Sure, I have no problem about the title bar. Since we cannot unload roms it's probably easiest to just do that in LoadROM. As for DWM functions, I did the same with the Win7 jumplist functions.

**Comment by bearoso** [comment]
I put the version number back in. I think it's ugly, but I totally agree with the reasoning.

I noticed the Blargg NTSC filters are extremely slow with OpenGL PBOs on Windows. This is because it maps the buffer and then does lots of little reads and writes in extending it to double height. This is a memory-mapped buffer, not memory, so every read/write succumbs to DMA overhead. I'm going to make it use the copy in root/filters and the functions I use with the GTK port.

**Comment by bearoso** [comment]
Ok, so I followed @qwertymodo's advice and changed the smp clock to 1024000 with a respective output rate of 32000Hz. This broke Circuit USA's timing hack, so I bumped it up.
I found a couple spots where I could optimize the SMP, so we're saving at least 1 if per opcode. Let me know if this breaks anything. We should be pretty close in performance to the Blargg SMP now.

**Comment by bearoso** [comment]
I've updated some of the documents. @OV2, would you have a look at them to make sure they're OK? Is there anything else holding up a release?

**Comment by OV2** [comment]
Looks good, hopefully someone actually reads the readme ;)
I've added a small hint about SPIRV-Cross to the compile instructions - if someone really downloads everything manually they also need that.

One thing I noticed when looking at the porting changes: should we just remove the lag_ms parameter from init? We already removed the finalize function, so it already requres some changes for existing ports.

I think we can release it. There were some additional hotkey requests in the forum, but that can also wait.
